samples/snippets/feature_store_service/batch_create_features_sample.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_batch_create_features_sample]
+from google.cloud import aiplatform_v1beta1 as aiplatform
+
+
+def batch_create_features_sample(
+    project: str,
+    featurestore_id: str,
+    entity_type_id: str,
+    requests: str,
+    location: str = "us-central1",
+    api_endpoint: str = "us-central1-aiplatform.googleapis.com",
+    timeout: int = 300,
+):
+    # The AI Platform services require regional API endpoints.
+    client_options = {"api_endpoint": api_endpoint}
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform.FeaturestoreServiceClient(client_options=client_options)
+    parent = f"projects/{project}/locations/{location}/featurestores/{featurestore_id}/entityTypes/{entity_type_id}"
+    batch_create_features_request = {"parent": parent, "requests": requests}
+    lro_response = client.batch_create_features(request=batch_create_features_request)
+    print("Long running operation:", lro_response.operation.name)
```

Comment:
nit. Do we need this print out? I saw assertion for the other print, but not this one. 

On the other hand, does it make sense to move the `# [END aiplatform_batch_create_features_sample]`  before the print out, just to keep the sample contain the essential line only.

Reply:
I agree it's a bit lengthy but the print() and the END mark are to be consistent to other aiplatform snippets, for example:
https://github.com/googleapis/python-aiplatform/blob/main/samples/snippets/dataset_service/create_dataset_tabular_bigquery_sample.py#L42-L46